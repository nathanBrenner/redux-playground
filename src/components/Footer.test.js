import React from "react";
import Footer from "./Footer";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router";

describe("Footer", () => {
  test("renders without crashing", () => {
    shallow(<Footer />);
  });

  test("snapshot", () => {
    const component = renderer.create(
      <MemoryRouter>
        <Footer />
      </MemoryRouter>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
