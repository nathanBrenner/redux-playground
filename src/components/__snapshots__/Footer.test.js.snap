// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<p>
  Show: 
  <a
    aria-current="true"
    className="active"
    href="/"
    onClick={[Function]}
    style={
      Object {
        "color": "black",
        "textDecoration": "none",
      }
    }
  >
    All
  </a>
  , 
  <a
    aria-current={false}
    className={undefined}
    href="/active"
    onClick={[Function]}
    style={undefined}
  >
    Active
  </a>
  , 
  <a
    aria-current={false}
    className={undefined}
    href="/completed"
    onClick={[Function]}
    style={undefined}
  >
    Completed
  </a>
</p>
`;
